Lib/test/regrtest.py

```diff
@@ -438,6 +438,8 @@ def main(tests=None, testdir=None, verbose=0, quiet=False,
 
     if slaveargs is not None:
         args, kwargs = json.loads(slaveargs)
```

Comment:
I'm not interested to backport this change. While regrtest of 3.5 is old, regrtest of 2.7 is even older. Backporting regrtest enhancements is more and more complex :-/

If we want to get more new regrtest features, I would prefer to "simply" copy regrtest from master to other branches.